template/Makefile

```diff
@@ -67,25 +69,40 @@ resources/app/%.go: $(DESIGN) | $(GOAGEN)
 resources/%.go: $(DESIGN) | $(GOAGEN)
 	@$(GOAGEN_CMD) controller --regen --pkg resources -o resources -d $(DESIGNPKG)
 
+ifndef GINKGO
```

Comment:
We need a semantically better way of figuring out if we should run in a container or not.